### PR TITLE
DT-1373: 🎨 Switch manage dps users to nunjucks

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: backend/controllers/addGroup.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const selectGroupFactory = (getUserAndGroups, saveGroup, reactUrl, manageUrl, ma
     const staffUrl = `${manageUrl}/${username}`
 
     try {
-      const [assignableGroups, user, userGroups] = await getUserAndGroups(res.locals, username)
+      const [user, assignableGroups, userGroups] = await getUserAndGroups(res.locals, username)
```
Comment (petergphillips) [inline backend/controllers/addGroup.js:14]: changed order as was confusing

File: backend/controllers/addRole.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const selectRolesFactory = (getUserAndRoles, saveRoles, reactUrl, manageUrl, mai
     const hasAdminRole = Boolean(res.locals && res.locals.user && res.locals.user.maintainAccessAdmin)
 
     try {
-      const [assignableRoles, user] = await getUserAndRoles(res.locals, username, hasAdminRole)
+      const [user, assignableRoles] = await getUserAndRoles(res.locals, username, hasAdminRole)
```
Comment (petergphillips) [inline backend/controllers/addRole.js:25]: changed order as was confusing

File: backend/controllers/userDetails.js [hunk 3]
```diff
@@ -9,15 +9,17 @@ const userDetailsFactory = (
   reactUrl,
   manageUrl,
   maintainTitle,
+  showExtraUserDetails,
```
Comment (petergphillips) [inline backend/controllers/userDetails.js:12]: page is now shared between external and DPS users, so need to toggle bits of the page off for now

File: manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/pages/StaffRoleProfilePage.groovy [hunk 4]
```diff
@@ -3,23 +3,20 @@ package uk.gov.justice.digital.hmpps.manageusers.pages
...
         browser.currentUrl.contains(url)
-        headingText.contains('roles')
     }
 
     static content = {
         headingText { $('h1').first().text() }
         removeButtons { $('.removeButton') }
         backLink { $('a.backlink')}
-        addButton { $('#add-button')}
+        addButton { $("[data-qa='add-role-button']")}
```
Comment (petergphillips) [inline manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/pages/StaffRoleProfilePage.groovy:16]: cope with page now in nunjucks

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**